src/sif/autocost.cc

```diff
@@ -341,6 +348,15 @@ AutoCost::AutoCost(const CostingOptions& costing_options, uint32_t access_mask)
   toll_factor_ = use_tolls < 0.5f ? (4.0f - 8 * use_tolls) : // ranges from 4 to 0
                      (0.5f - use_tolls) * 0.03f;             // ranges from 0 to -0.15
 
+  // Preference to use tracks. Is a value from 0 to 1.
+  float use_tracks = costing_options.use_tracks();
```

Comment:
I'd rather use boolean to guarantee that tracks will be excluded from the route. But in this case we will fail in cases where tracks is the only option to complete the route.